src/app/content/studyGuides/actions.ts

```diff
@@ -1,11 +1,16 @@
-import { HighlightsSummary } from '@openstax/highlighter/dist/api';
 import { createStandardAction } from 'typesafe-actions';
-import { StudyGuidesHighlights } from './types';
+import { CountsPerSource, HighlightData, SummaryHighlights, SummaryHighlightsPagination } from '../types';
 
-export const receiveStudyGuides = createStandardAction('Content/StudyGuides/receive')<HighlightsSummary>();
+export const receiveSummaryStudyGuides = createStandardAction(
+  'Content/StudyGuides/Summary/receive'
+)<SummaryHighlights, SummaryHighlightsPagination>();
 export const openStudyGuides = createStandardAction('Content/StudyGuides/Summary/open')<void>();
+export const initializeStudyGuidesSummary = createStandardAction('Content/StudyGuides/Summary/Initialize')<void>();
```

Comment:
this initialize action isn't being used, i think in MH this is dispatched when it mounts and all initialization is done then, but you're doing that with a hook into the open action

idk what the correct approach is, but something seems to need to be cleaned up here